src/Home.js

```diff
@@ -1,45 +1,51 @@
 import { GiSpiderWeb } from "react-icons/gi";
-import { useState } from "react";
+import { useEffect, useState } from "react";
+import Creatorlist from "./Components/Creatorlist";
 
 const Home = () => {
 
-    const [Creator, setCreator]= useState(
-        [
-            {id:1, author: "Tom", title: "How to work"},
-            {id:2,  author: "John", title: "Learning how to code" },
-            {id:3,  author: "Andrew", title: "Achieving your goals"},
-
-        ]
-    );
+    const [creator, setCreator]= useState(null);
+    const[error, setError]= useState(null);
+    const[loading, setLoading]=useState(true);
+
+const[course, setCourse]= useState('open');
+
+    useEffect(()=>{
+      setTimeout(()=>{
+        fetch('http://localhost:8000/creators')
+        .then ((Response)=>{
+            if(!Response.ok){
+                throw Error('server is busy. Please try again')
+            }
+         return Response.json()
+        })
+        .then((creator)=>{
+        setCreator(creator)
+        setLoading(false)
+        })
+        .catch((err)=>{
+        console.log(err.messsage)
+        setError(err.message)
+        setLoading(false)
+    })
+      },1000)
+
+},[]);
 
     return (  
         <div classname="home">
             <h1
```

Comment:
Hi @Techtrial101, Your app is running fine and I give you kudos for the work done so far. 

However, there is a small error in the console. the case we use for className in react is not what you used here. please fix it.